frontend/app/views/spree/address/_form.html.erb

```diff
@@ -65,30 +67,31 @@
             class: !have_states ? 'required' : '',
             style: have_states ? 'display: none;' : '',
             disabled: have_states,
-            autocomplete: address_type + ' address-level1'
+            autocomplete: address_type + ' address-level1',
+            required: true
```

Comment:
The `required` here should maybe follow the same logic as the `required` class. `!have_states ? 'required' : ''`.

Do we maybe need to remove the `required` class from those inputs as well? And use maybe the `required` attribute as a selector in the css if there's need.